sensu/plugins/check-glance-store.py

```diff
@@ -16,10 +16,12 @@
 from keystoneclient.v2_0 import client as ksclient
 
 def is_remote_image(image):
-    if image.get('location'):
-        http_re = re.compile(r'^(http|https|ftp)://')
-        return http_re.match(image.location)
-    return False
+    if image.get('locations'):
+        for location in image.locations:
+            http_re = re.compile(r'^(file)://')
```

Comment:
Just for a tiny efficiency sake, you could define the regex outside of the loop. That way it's defined once and used throughout the loop rather than re-defined each time the loop runs.